### PR TITLE
Automated styling update from server

### PR DESCRIPTION
**Pull request by nextcloud-command**
Auto-generated update of our locally cached css styling from
 * https://nextcloud.github.io/server/apps/theming/css/default.css
 * https://nextcloud.github.io/server/core/css/server.css
 * https://nextcloud.github.io/server/dist/icons.css

### CODE REVIEW

**Review by Antreesy — APPROVED** [review]

File: src/components/NcInputField/NcInputField.vue [hunk 1]
```diff
@@ -441,7 +441,7 @@ export default {
 		margin-inline: var(--input-padding-start) var(--input-padding-end);
 		max-width: fit-content;
 		font-size: var(--input-label-font-size);
-		inset-block-start: calc((var(--default-clickable-area) - var(--default-line-height)) / 2); // center the label vertically
+		inset-block-start: calc((var(--default-clickable-area) - 1lh) / 2); // center the label vertically
```
Comment (ShGKme) [inline src/components/NcInputField/NcInputField.vue:444]: This is not `Automated styling update from server` 👀

**Review by ShGKme — COMMENTED** [review]
My bad, I missed that `lh` is not supported by Firefox

File: src/components/NcInputField/NcInputField.vue (hunk 1 above)
Comment (Antreesy) [reply src/components/NcInputField/NcInputField.vue]: I can move it to separate PR, but it'll take more efforts

Comment (Antreesy) [reply src/components/NcInputField/NcInputField.vue]: and dark.css isn't updated for some reason..

File: src/components/NcInputField/NcInputField.vue [hunk 2]
```diff
@@ -441,7 +441,7 @@ export default {
 		margin-inline: var(--input-padding-start) var(--input-padding-end);
 		max-width: fit-content;
 		font-size: var(--input-label-font-size);
-		inset-block-start: calc((var(--default-clickable-area) - var(--default-line-height)) / 2); // center the label vertically
+		inset-block-start: calc((var(--default-clickable-area) - var(--default-line-height) * var(--input-label-font-size)) / 2); // center the label vertically
```
Comment (ShGKme) [inline src/components/NcInputField/NcInputField.vue:444]: It is not compatible with Nextcloud 28..29 with unit line height

**Comment by ShGKme** [comment]
> My bad, I missed that `lh` is not supported by Firefox

Oops, I missed with lh function.

Not very well supported, but works in FF

https://caniuse.com/mdn-css_types_length_lh

File: src/components/NcInputField/NcInputField.vue (hunk 1 above)
Comment (ShGKme) [reply src/components/NcInputField/NcInputField.vue]: > and dark.css isn't updated for some reason..

Yes, it is not automated. Same with Talk Desktop — default styles are available from source, but theming required a PHP server

File: src/components/NcInputField/NcInputField.vue (hunk 2 above)
Comment (Antreesy) [reply src/components/NcInputField/NcInputField.vue:444]: Couldn't find a way to transform it to another unit, but as `lh` is working, shouldn't be a problem?

**Review by ShGKme — APPROVED** [review]

**Comment by marcoambrosini** [comment]
Cypress still failing

**Review by marcoambrosini — APPROVED** [review]

**Pull request merged**